jms/1.7/modules/ROOT/pages/jms-studio-configuration.adoc

```diff
@@ -86,7 +86,7 @@ image::jms-operations.png[Select any of the JMS Connector Operations from the Mu
 [[configure-global-element]]
 == Configure a Global Element for the Connector
 
-When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. JMS Connector comes with a fine-tuned set of default values, both for publishing and consuming messages. The only requirement is to configure which connection to use. You can define a global default behavior for all the operations associated with the configuration and then override each parameter in only the operations that require a custom behavior.
+To avoid connectivity issues, create separate JMS configurations for publishing and consuming, instead of creating a shared JMS configuration for both publishing and consuming. JMS Connector comes with a fine-tuned set of default values, both for publishing and consuming messages. The only requirement is to configure which connection to use. You can define a global default behavior for all the operations associated with the configuration and then override each parameter in only the operations that require a custom behavior.
```

Comment:
A minor point, the new wording is inconsistent with the following sentence. It might be better to change the second one to "for both publishing and consuming". This also begs the question on why say "both" in either case, since and implies both.

Reply:
Thanks Beth! What if I change the second sentence to:
JMS Connector comes with a fine-tuned set of default values for publishing and consuming messages.

Reply:
Looks good to me :)